src/main/resources/embed-tag-rules.json

```diff
@@ -3,15 +3,18 @@
     "h5p": {
       "required": [
         "data-resource",
-        "data-url"]
+        "data-url"],
+      "optional": [
```

Comment:
Er det noe grunn til at `optional` har elementene i en nested array, mens `required` har bare vanlig array med elementer?

Reply:
Ane ikkje. Det var sånn, eg har berre lagt til nye felt.

Reply:
As long as it works i guess, ser greit ut ellers.